Cargo.toml

```diff
@@ -14,7 +14,7 @@ categories = ["development-tools::debugging", "embedded", "emulators", "network-
 exclude = ["examples/**/*.elf", "examples/**/*.o"]
 
 [dependencies]
-bitflags = "1.2.1"
+bitflags = "1.3"
```

Comment:
nit: bitflags 1.3.0 was yanked. might as well be explicit, and depend on 1.3.1 instead

Reply:
Won't `cargo` select it automatically?

Reply:
most likely, yes, unless someone just-so-happened to explicitly specify 1.3.0, which would be very unlikely.

on second thought, we can just leave it as is.